### PR TITLE
Create table animals

### PR DESCRIPTION
**Pull request by Mucyosoda**
**Create database Vet_clinic**
Database vet_clinic has been created with one table, and a table with 5columns (id, name, date_of_birth, escape_attemps, neutered, weight_kg) and inserted 4 rows. 
Different files are added:
- data.sql
- schema.sql
- queries.sql

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,3 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
 
 ## Getting Started
```
Comment (Meltrust) [inline README.md:6]: - [ ] Please, kindly add the title and description of the project in your readme file:

image

Just add and customize the title and description shown on the template to reflect the information of your project 💪

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered) VALUES ('Agumon', date '2020-02-03', 10.23, true);
```
Comment (Meltrust) [inline data.sql]: - [ ] Agumon is missing "escape attempts".  Please, add the value (should be explicitly zero). 

Example: 

```SQL
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Mucyosoda,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice code organization and indentation✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (Mucyosoda) [reply data.sql]: Thanks a lot

**Review by Meltrust — APPROVED** [review]
Hi @Mucyosoda,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**